test/integration_tests/test_config_reader.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+import json
+from src.config_reader import ConfigReader
+
+
+class TestConfigReader(unittest.TestCase):
```

Comment:
Could you perhaps indicate on top of each of your tests explicitly indicate which of the 4 ML testing categories (see slides) they belong to, to make it clear we covered them all? i.e. in this case, it would be "Tests for ML infrastructure" for all of these since you are integration testing the ML pipeline.

Reply:
sure